preupg/utils.py

```diff
@@ -762,6 +762,41 @@ def update_variables(self, result_dir, result_name, xml_result_name, html_result
         self.content = content
 
     @staticmethod
+    def get_oscap_version():
+        """
+        Get version of the OpenSCAP.
+
+        Return version of the openscap as dict with the 'major', 'minor' and
+        'patch' keys.
+
+        Raise OSError when the oscap utility is not installed.
+        """
+        oscap_version = {"major" : -1, "minor" :  -1, "patch" : -1}
+        def _process_oscap_output(oscap_msg):
+            if oscap_version["major"] >= 0:
+                return
+            # this should happen only in case of first line
```

Comment:
This comments seems a bit confusing to me? Did you want to say that the version of OpenSCAP appears on the first line? Please reword the sentence.

Reply:
Exactly :-)